ovs/extensions/healthcheck/utils/helper.py

```diff
@@ -45,7 +45,6 @@ class Helper(object):
     rights_dirs = settings["healthcheck"]["rights_dirs"]
     owners_files = settings["healthcheck"]["owners_files"]
     check_logs = settings["healthcheck"]["check_logs"]
-    client = SSHClient('127.0.0.1', username='root')
```

Comment:
SSHClient as class variable killed the system.
Localhost in the SSHClient is bad practise too.

Good that this has been resolved.